www/templates/main-goals.html

```diff
@@ -0,0 +1,68 @@
+<ion-view view-title="Goals">
+	<ion-nav-buttons side="right">
+    	<button class="button button-icon" ng-click="openModal()">
+        	<i class="icon ion-compose"></i>
+   	 	</button>
+   	 </ion-nav-buttons>
+     <!--ion-nav-buttons side="left">
+      <button class="button button-icon icon ion-ios-minus-outline" ng-click="showDelete = !showDelete"></button>
+     </ion-nav-buttons-->
+<ion-content class="has-header" overflow-scroll="true" padding="true">
+	 </div>
+      <!--div>
+        <img ng-src="https://habitica-assets.s3.amazonaws.com/avatars/4f369eef-aed4-4408-bcbf-b34896daf7e3.png">
+      </div-->
+      <div>
+        <a class="item item-thumbnail-left">
+          <img ng-src="https://www.habitica.com/export/avatar-{{user.data._id}}.png">
+          <div id="stats-wrapper">
+            <i id="habitica-heart-icon" class="icon ion-heart"></i>
+            <progress value="50" max="50"></progress>
+            <div class="stats-value-wrapper">
+              <p class="stats-value">50/50</p>
+              <p class="stats-name">Health</p>
+            </div>
+          </div>
+          <div>
+            <i id="habitica-star-icon" class="icon ion-android-star"></i>
+            <progress class="golden" value="{{user.data.stats.exp}}" max="2060"></progress>
+            <div class="stats-value-wrapper">
+              <p class="stats-value">{{user.data.stats.exp}}/2060</p>
+              <p class="stats-name" style="margin-top: 1.5px;">Exprerience</p>
+            </div>
+          </div>
+        </a>
+      </div>
+        <div class="center" style="margin-top:-5px;">
+          <img src="img/ic_header_healer.png" style="height:40px; width:40px;" class="center">
+          <h2 style="font-size:12px; text-align:center; margin:0px">{{user.data.profile.name}}</h2>
+          <p style="font-size:10px; text-align:center;">{{user.data.stats.class}}</p>
+        </div>
+      <ion-list can-swipe="true">
+        <ion-item class="item" ng-repeat="goal in goals" style="border:1px;">
+          <div class="row">
+          <lable class="checkbox">
+              <input type="checkbox" ng-click="completeGoal(goal)">
+          </lable>
+          <!--div class="col col-20" style="display:flex; justify-content:flex-end; align-items:center;">
+          <button class="button button-clear" style="font-size:2rem !important">
+            <i icon="ion-checkmark-circled"></i>
+          </button>
+          </div-->
+          <div class="col col-80" style="padding-left: 5% !important;">
+          <h2>{{goal.name}}</h2>
+          <p style="font-size: 10px">{{goal.note}}</p>
+          </div>
+          <ion-option-button class="button-assertive" ng-click="removeGoal(goal)">Delete
```

Comment:
I'm a bit confused. I assumed that you would use.
Is there a reason why you would handle it on your own?
I would prefer using the standard directives as much as possible because ionic is likely to make additional improvements going forward...

`<ion-list show-delete="shouldShowDelete">
   <ion-item>
  ...
    <ion-delete-button class="ion-minus-circled"
                       ng-click="removeGoal(goal)">
    </ion-delete-button>
   </ion-item>
</ion-list>
```